liferay-gradle/blade.controlmenuentry/build.gradle

```diff
@@ -1,5 +1,5 @@
 dependencies {
-	compile group: "com.liferay", name: "com.liferay.product.navigation.control.menu.api", version: "2.0.0"
+	compile group: "com.liferay", name: "com.liferay.product.navigation.control.menu.api", version: "[3.0.0-SNAPSHOT,3.0.0)"
```

Comment:
The Jar is published now, I just added another commit removing the usage of the snapshot.